package.json

```diff
@@ -82,7 +87,8 @@
         "vue-class-component": "^6.1.1",
         "vue-property-decorator": "^6.0.0",
         "vue-template-compiler": "^2.5.9",
-        "vue-template-loader": "^0.3.1"
+        "vue-template-loader": "^0.3.1",
+        "webpack-bundle-analyzer": "^2.13.1"
```

Comment:
Should it be inside the devDependencies ?

Reply:
Will be fixed